locales/en/kafka.json

```diff
@@ -109,6 +109,13 @@
   "partitions": "Partitions",
   "partitions_value": "up to 500",
   "select_filter_category": "Select filter category",
+  "settings": {
+    "connection_re_authentication_label": "Connection re-authentication:",
```

Comment:
Should the ":" be included as part of this text string? I'm not sure if there's any benefit to the translator. We also should reuse this label for an aria-label where I wouldn't expect to include punctuation like this.